bot/seasons/evergreen/issues.py

```diff
@@ -0,0 +1,41 @@
+import logging
+
+import discord
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+
+class Issues(commands.Cog):
+    """Cog that allows users to retrieve issues from GitHub."""
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(aliases=("issues",))
+    async def issue(self, ctx, number: int, repository: str = "seasonalbot", user: str = "python-discord"):
+        """Command to retrieve issues from a GitHub repository."""
+        url = f"https://api.github.com/repos/{user}/{repository}/issues/{str(number)}"
+        status = {"404": f"Issue #{str(number)} doesn't exist in the repository {user}/{repository}.",
+                  "403": f"Rate limit exceeded. Please wait a while before trying again!"}
```

Comment:
Would look better formatted as 
```py
status = {
    404: "...",
    403: "...",
}
```

And yes, they should be integers instead of strings as line 26 is doing as `r.status` is initially an integer anyway as you can see by the string conversion being done on lines 25 and 26.